source/settings/settings-reference.txt

```diff
@@ -0,0 +1,122 @@
+.. _compass-interface-settings:
+.. _compass-settings-reference:
+
+--------------------------
+Compass Interface Settings
+--------------------------
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can customize |compass| to meet your specific needs on the |compass-short| 
+interface. Some options are only available in the :guilabel:`Settings` panel.
+
+If you have already configured |compass-short| settings on the command line or 
+in a configuration file, you cannot modify those settings in the |compass-short| 
+interface.
+
+Settings Panel Location
+-----------------------
+
+To open the |compass-short| :guilabel:`Settings` panel, navigate to the 
+:guilabel:`MongoDB Compass` system menu and select :guilabel:`Settings`.
+
+.. figure:: /images/compass/settings-panel-location.png
+  :alt: Settings panel location under the MongoDB Compass system menu
+
+Alternatively, you can use keyboard shortcuts to open the :guilabel:`Settings` 
+panel: 
+
+- Windows / Linux: :guilabel:`Ctrl + ,`
```

Comment:
[suggestion]

With the current presentation, it looks like the plus sign is part of the series of keys that needs to be pressed. What if we tried:

```
- Windows / Linux: ``Ctrl`` + ``,``
```

Hard to say if this will be better without seeing it staged, but it might be clearer that the plus sign isn't actually meant to be pressed here.

Reply:
Yeah I was going back and forth with this one. I think it's because the ``,`` makes it look weird and almost incomplete. The point you made about it being potentially confusing makes sense, though, so I think we'll opt for the monospacing like you suggested here.